src/main/java/no/nav/pto/veilarbportefolje/admin/v2/AdminV2Controller.java

```diff
@@ -0,0 +1,53 @@
+package no.nav.pto.veilarbportefolje.admin.v2;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.auth.context.AuthContextHolder;
+import no.nav.common.types.identer.AktorId;
+import no.nav.common.utils.EnvironmentUtils;
+import no.nav.pto.veilarbportefolje.auth.DownstreamApi;
+import no.nav.pto.veilarbportefolje.domene.AktorClient;
+import no.nav.pto.veilarbportefolje.opensearch.OpensearchIndexer;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import static no.nav.pto.veilarbportefolje.auth.AuthUtils.erSystemkallFraAzureAd;
+import static no.nav.pto.veilarbportefolje.auth.AuthUtils.hentApplikasjonFraContex;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v2/admin")
+@RequiredArgsConstructor
+public class AdminV2Controller {
+	private final String PTO_ADMIN = new DownstreamApi(EnvironmentUtils.isProduction().orElse(false) ?
+			"prod-fss" : "dev-fss", "pto", "pto-admin").toString();
+	private final AktorClient aktorClient;
+	private final OpensearchIndexer opensearchIndexer;
+	private final AuthContextHolder authContextHolder;
+
+	@PutMapping("/indeks/bruker/fnr")
+	public String indeks(@RequestBody AdminIndeksBrukerRequest adminIndeksBrukerRequest) {
+		sjekkTilgangTilAdmin();
+		String aktorId = aktorClient.hentAktorId(adminIndeksBrukerRequest.fnr()).get();
```

Comment:
Jeg tror vi nå mister en validering av fnr som tidligere ble gjort vha. `Fnr.ofValidFnr(fnr)`. Dette kan potensielt lede til at vi sender ugyldige fnr til AktorClient, som igjen muligens kan lede til flere feil.

Selv om jeg tror sannsynligheten er lav for at vi har ugyldige fnr så kan det sikkert være greit å stoppe det så tidlig som mulig likevel.